### PR TITLE
feat: max price per series

### PR DESCRIPTION
**Pull request by jastheace**
### Description

N/A - max price is for single product; this makes it by product

Updates max price so it is product specific instead of being a single number

Jase/Myanth#7701

### Testing

untested but works in theory; don't have an endpoint that breaks the price currently

changes maxPrice to maxPrice3070, maxPrice3080, maxPrice3090

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -141,7 +141,9 @@ const proxy = {
 
 const store = {
 	country: envOrString(process.env.COUNTRY, 'usa'),
-	maxPrice: envOrNumber(process.env.MAX_PRICE),
+	maxPrice3070: envOrNumber(process.env.MAX_PRICE_3070),
+	maxPrice3080: envOrNumber(process.env.MAX_PRICE_3080),
+	maxPrice3090: envOrNumber(process.env.MAX_PRICE_3090),
```
Comment (jef) [inline src/config.ts:146]: Users will need to know this is breaking change for `MAX_PRICE` though. Perhaps we should include a warning for now and then we can delete later.

```typescript
if (process.env.MAX_PRICE) logger.warn('ℹ MAX_PRICE is deprecated, please use MAX_PRICE_${series}')
```

**Review by jef — CHANGES_REQUESTED** [review]
This makes a lot of sense to me. Just one comment on reporting to the user of breaking change should be sufficient.

Thanks!

File: src/config.ts (hunk 1 above)
Comment (jastheace) [reply src/config.ts:146]: added relevant section to index with comments so other deprecated items can be added in the pre-execution

**Review by jef — APPROVED** [review]
Great! Thank you very much for cleaning this up and adding the message.

**Pull request merged**